Ednpoints.txt

```diff
@@ -0,0 +1,132 @@
+The same routes for users, chefs and delivery
+POST: /auth/register - registration
+POST: /auth/login - enter to account
+GET: /auth/logout - exit from account
+POST: /auth/forgot-password - request to change password
+POST: /auth/reset-password - password reset
+
+/************/
+
+USER ENDPOINTS:
+
+GET: /users/:userId - get user's info
+PATCH: /users/:userId - update user's info
+DELETE: /users/:userId - delete user's account
+
+GET: /dishes - get a list of dishes
+GET: /dishes/:dishId - get dish information
+GET: /dishes/popular - get popular dishes
+GET: /dishes/random - get some random dish
+
+GET: /chefs - get all chefs
+
+GET: /users/:userId/favourites?type=chef(dish) - get user's favourite dishes or chefs
+	or /users/:userId/favourite-dishes and /users/:userId/favourite-chefs
+POST, DELETE:
+/users/:userId/favourites?type=dishes - add/delete the dish to favourites (passing dishId)
+/users//:userId/favourites?type=chefs - add/delete the chef to favourites (passing chefId)
+
+**user's cart will be stored on the server instead of in local storage
+GET: /users/:userId/cart - get the cart info
+POST: /users/:userId/cart - add the dish to the cart (passing dishId)
+PATCH: /users/:userId/cart - update information about the dish in the cart (passing dishId)
+DELETE: /users/:userId/cart  - delete the dish from the cart (passing dishId)
+
+GET: /notifications/:userId - get notifications of this user
```

Comment:
do you want to store it in the database? we can have millions of documents there
on my opinion, notification should be sent and forgotten forever, it's necessary at the moment only

Reply:
Great point,  we didn't think about the fact that this is meaningless data, it doesn't need to be stored. Okay, but in that case how do we implement storing notifications and displaying them continuously from all devices until the user reads them?

Reply:
we can use `localStorage` for this and store notifications in JSON format there until user mark a notification as read
also we can use other platforms to notify users, Telegram API or email for example and have internal notifications as a sideway